.github/workflows/nodejsci.yml

```
@@ -6,6 +6,10 @@ on:
     paths:
     - 'packages/**'
 
+permissons:
```

actionlint (1.7.12)

[error] unexpected key "permissons" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)